components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/ServerApplicationManagementService.java

```diff
@@ -429,6 +449,100 @@ public String exportApplication(String applicationId, Boolean exportSecrets) {
         }
     }
 
+    /**
+     * Export an application identified by the applicationId, in the given format.
+     *
+     * @param fileType      The format of the exported string.
+     * @param applicationId ID of the application to be exported.
+     * @param exportSecrets If True, all hashed or encrypted secrets will also be exported.
+     * @return string of the application in the given format.
+     */
+    public TransferResource exportApplicationAsFile(String fileType, String applicationId, Boolean exportSecrets) {
```

Comment:
Shall we use the same parameter order as in other `exportApplicationAsFile` methods for consistency. (applicationId, exportSecrets, fileType)

Reply:
Changed parameter order for all exportApplicationAsFile named methods to -> applicationId, exportSecrets, fileType